impeller/entity/contents/runtime_effect_contents.cc

```diff
@@ -47,6 +47,10 @@ bool RuntimeEffectContents::Render(const ContentContext& renderer,
   auto context = renderer.GetContext();
   auto library = context->GetShaderLibrary();
 
+#ifdef FML_OS_ANDROID
```

Comment:
Can you just move this to the top and add a comment that this is a no-op while we wire things up?

Reply:
Done